docs/usage/flower17.rst

```diff
@@ -0,0 +1,107 @@
+FLOWER17 dataset classification using OpenCV and Scikit-Learn
+=============================================================
+
+This example will show you how to train and test a model on the `FLOWER17` dataset.
+This dataset contains 17 classes of flower species, each having 80 images.
+
+Download the dataset and verify with with ``sha256sum``.
+
+.. literalinclude:: /../examples/flower17/dataset.sh
+
+.. code-block:: console
+
+    17flowers.tgz: OK
+
+The model we'll be using is the `RandomForestClassifier` which is a part of
+``dffml-model-scikit``, a DFFML plugin which allows you to use Scikit-Learn via DFFML.
+We'll also be using the various operations from ``dffml-operations-image``, which is
+a DFFML plugin which allows you to do image processing via DFFML.
+We can install them with ``pip``.
+
+.. code-block:: console
+
+    $ pip install -U dffml-model-scikit dffml-operations-image
+
+Extract the dataset.
+
+.. code-block:: console
+
+    $ tar -xzf 17flowers.tgz
+
+All the images are now in a folder called `jpg`.
+Let's split these images into flower_dataset/train and flower_dataset/test directories,
+each directory containing sub-directories corresponding to the 17 flower classes using split.py.
+
+.. literalinclude:: /../examples/flower17/split.py
+
+.. code-block:: console
+
+    $ python split.py
+
+The train directory contains 70 images for each class while the test directory contains
+10 images for each class.
+
+Create a dataflow config file which will be used by the
+:py:class:`DataFlowSource <dffml.source.df.DataFlowSource>` to preprocess
+the data before feeding it to the model.
```

Comment:
After this paragraph explain what preprocessing needs to be done, and why. Link to the image operations. In another PR, make sure we have docstrings for all of them explaining what they do. Finally link to the image operations using

```rst
:ref:`plugin_operation_dffml`
```

Reply:
Also add to `scripts/docs/care`